hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -184,8 +200,8 @@ class ExampleProperty:
     to calculate these properties.
     """
 
-    def __init__(self, examples):
-        self.example_stack = []
+    def __init__(self, examples: "Examples"):
+        self.example_stack: "List[int]" = []
```

Comment:
I'd prefer unquoted annotations, where those are valid.

Reply:
I use quoted annotations in 2 places generally:
- Forward references
- In function bodies for performance reasons (`Optional[List[Tuple[float, str]]]]` is a lot of function calls)
How do you feel about the latter?

Reply:
Hmm, with that justification I'm OK with quotes; conjecture is one of the rare actually-perf-sensitive parts of Hypothesis.